scripts/parse_specs.py

```diff
@@ -914,6 +969,9 @@ def parse(section, version, tags, meta, ref):
             })
 
     specs = sorted(specs, key=lambda s: s['header']['ordinal'])
+    _inline_extended_structs(specs, meta)
+    registry = [f for f in files if f.endswith('registry.yml')][0]
+    _generate_structure_type_t(specs, meta, registry)
```

Comment:
I'm not sure if I understand why this can't simply be a part of the `generate_registry` function. Right now it's called almost immediately after.

Reply:
Perhaps an artifact of a previous iteration of this change when it was generating a separate file?

Reply:
To generate the structure type we need to inline the extended structs, which happens after `generate_registry`, but since registry does not depend on this we could run these steps together after `_inline_extended_structs`